src/Goerli-DssSpell.sol

```diff
@@ -16,29 +16,79 @@
+
 interface RwaUrnLike {
     function draw(uint256) external;
 }
 
+interface CureLike {
+    function lift(address) external;
+}
+
+interface TeleportJoinLike {
+    function file(bytes32,bytes32,address) external;
+    function file(bytes32,bytes32,uint256) external;
+    function vow() external view returns (address);
+    function vat() external view returns (address);
+    function daiJoin() external view returns (address);
+    function ilk() external view returns (bytes32);
+    function domain() external view returns (bytes32);
+}
+
+interface TeleportRouterLike {
+    function file(bytes32,bytes32,address) external;
+    function gateways(bytes32) external view returns (address);
+    function domains(address) external view returns (bytes32);
+    function dai() external view returns (address);
+}
+
+interface TeleportOracleAuthLike {
+    function file(bytes32,uint256) external;
+    function addSigners(address[] calldata) external;
+    function teleportJoin() external view returns (address);
+}
+
+interface EscrowLike {
+    function approve(address,address,uint256) external;
+}
+
 contract DssSpellAction is DssAction, DssSpellCollateralAction {
 
     // Provides a descriptive tag for bot consumption
     string public constant override description = "Goerli Spell";
 
-    uint256 constant internal MILLION  = 10 **  6;
-    uint256 constant internal BILLION  = 10 **  9;
-    uint256 constant internal WAD      = 10 ** 18;
+    address internal constant TELEPORT_JOIN = 0xAaEf3f1523291aDdaF98a8535c27F25971d823D2;
+    address internal constant ORACLE_AUTH = 0xD40ab915cB8232E8188e1a9137E4b5dCB86F0fd8;
+    address internal constant ROUTER = 0x75c2D0A33cB245acD1F10798fEdD4a42Be4951a9;
+    address internal constant LINEAR_FEE = 0x72Cb460888D401f991AB1a78ffc48EFcDcd155e8;
+
+    bytes32 constant internal ILK = "TELEPORT-FW-A";
+    bytes32 constant internal DOMAIN_ETH = "ETH-GOER-A";
 
-    // address constant RWA008_A_OPERATOR       = 0x03f1A14A5b31e2f1751b6db368451dFCEA5A0439;
-    // address constant RWA008_A_OUTPUT_CONDUIT = 0x1aA21d2E39EC0da185CA04609c8868bC324d8553;
-    uint256 constant RWA009_DRAW_AMOUNT      = 25_000_000 * WAD;
+    bytes32 constant internal DOMAIN_OPT = "OPT-GOER-A";
+    address internal constant TELEPORT_GATEWAY_OPT = 0xe57e6b2eEEf91C068849bd6066d1041A00A4F654;
+    address internal constant ESCROW_OPT = 0xbc892A208705862273008B2Fb7D01E968be42653;
+    address internal constant DAI_BRIDGE_OPT = 0x05a388Db09C2D44ec0b00Ee188cD42365c42Df23;
+    address internal constant GOV_RELAY_OPT = 0xD9b2835A5bFC8bD5f54DB49707CF48101C66793a;
+
+    bytes32 constant internal DOMAIN_ARB = "ARB-GOER-A";
+    address internal constant TELEPORT_GATEWAY_ARB = 0x3F7Eea7c2D08bc6F249759082360E14c829b2A92;
+    address internal constant ESCROW_ARB = 0xDA10009cBd5D07dd0CeCc66161FC93D7c9000da1;
+    address internal constant DAI_BRIDGE_ARB = 0x467194771dAe2967Aef3ECbEDD3Bf9a310C76C65;
```

Comment:
This contract is missing the deny to deployer.

Reply:
The deployer hadn't yet been denied for the Goerli Arbitrum (slow) bridge contracts. It's done now.